### PR TITLE
Gives the lavaland ashwalker den the same treatment I gave the icebox one

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
image

## How This Contributes To The Skyrat Roleplay Experience
After I redid the icebox version, the lavaland version was suddenly severely lacking in mapping quality, well at least it used to.

## Changelog

:cl:
add: The ash walker den for lavaland has been remapped!
/:cl:

### CODE REVIEW

**Comment by Zergspower** [comment]
Losing the trophy room? :(,  and lanterns?

related, i never understood the alter as it kinda... does nothing?

**Comment by dawsonkeyes** [comment]
No ashie surgical bed, 10/10

**Comment by Paxilmaniac** [comment]
Replaced some of the walls with stone in case the ashies decide they want to build out

**Comment by YakumoChen** [comment]
Replacing the cult altar with the one that makes pants seems like a weird immersion take for ash walkers.

**Comment by Paxilmaniac** [comment]
the necropolis giveth pantaloons

**Comment by dawsonkeyes** [comment]
> Replacing the cult altar with the one that makes pants seems like a weird immersion take for ash walkers.

ash walkers are primitive tribal mystics, would a machine capable of forming `pants` from nothing not be an infinitely holy relic?

**Comment by Zergspower** [comment]
> > Replacing the cult altar with the one that makes pants seems like a weird immersion take for ash walkers.
> 
> ash walkers are primitive tribal mystics, would a machine capable of forming `pants` from nothing not be an infinitely holy relic?

because it makes absolutely no sense RP wise - it'd be better if it was a stick or something but shorts?

**Comment by dawsonkeyes** [comment]
> > > Replacing the cult altar with the one that makes pants seems like a weird immersion take for ash walkers.
> > 
> > 
> > ash walkers are primitive tribal mystics, would a machine capable of forming `pants` from nothing not be an infinitely holy relic?
> 
> because it makes absolutely no sense RP wise - it'd be better if it was a stick or something but shorts?

they're tribal warriors whose best grasp on clothing is a blanket and metal sheets. a pants altar would absolutely fit in as a site of holy importance

**Review by Zonespace27 — APPROVED** [review]

File: _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 /obj/structure/stone_tile/slab,
 /obj/item/storage/bag/quiver,
 /turf/open/misc/asteroid/basalt/lava_land_surface,
-/area/template_noop)
+/area/ruin/unpowered/ash_walkers)
```
Comment (Elli-Skala) [inline _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm:35]: Don't we have to leave it powered or else drying racks stop working?

Comment (Paxilmaniac) [reply _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm:35]: that's how the ashwalker base has always been, the drying rack already there I edited to just not use power

in reality they shouldn't use power at all but what can you do

**Comment by Zergspower** [comment]
image
i think it may be a good idea to seal these 3 gaps off like it was before given how plasma cutters go for miles (and in this case a miner decided to drag fauna through)

**Comment by Stalkeros2** [comment]
They **really** could use a welding torch and a fuel pool re-addition. The former is useful for crafting and construction (I mean that's obvious duh) whereas the latter is, -also quite obviously-, needed for welder refuels.
I am also fairly certain that the quiver they get in their village is the medieval CTF one, which spawn with the arrows that deal a LOT of damage and can't be acquired any other way)

**Comment by Rvali** [comment]
With the new changes to baking with a forge, the lavaland and the ice moon Ashie den do not have a baking tray to bake Rootdough with.

**Comment by Paxilmaniac** [comment]
I knew I was forgetting something with that, I'll give ashies some trays soon

**Review by GoldenAlpharex — APPROVED** [review]
Looks good, let's send it!

**Pull request merged**